packages/core/lib/testing/TestRunner.js

```diff
@@ -39,6 +39,14 @@ class TestRunner {
     this.TEST_TIMEOUT = (options.mocha && options.mocha.timeout) || 300000;
   }
 
+  disableChecks() {
```

Comment:
What would you think about renaming this option so as to make it clear that it is related to codec checks? Something like `disableCodecChecks` and so forth?

Reply:
Yeah, that would probably be clearer; I'll go do that.